File: fsharp/raindrops/Raindrops.fs
module Raindrops

let convert (number: int): string =
    [ (3, "Pling")
      (5, "Plang")
      (7, "Plong") ]
    |> List.filter (fun (v, _) -> number % v = 0)
    |> List.map snd
    |> function
    | l when l.Length = 0 -> string number
    | l -> System.String.Concat l

File: Exercism/csharp/raindrops/Raindrops.cs
using System.Collections.Generic;
using System.Linq;

public static class Raindrops
{
    private static readonly (int factor, string name)[] _drops = { (3, "Pling"), (5, "Plang"), (7, "Plong") };
    public static string Convert(int number)
    {
        var processed = _drops.Aggregate("", (s, i) => number % i.factor == 0 ? s + i.name : s);
        return processed.Length == 0 ? number.ToString() : processed;
    }
}